src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2444,7 +2444,12 @@
         <element name="code">
             <ref name="attlist.code"/> <!-- @id, @title, @xml:space, @xml:lang, @lang, @dir, @epub:type, @class -->
             <zeroOrMore>
-                <ref name="inline"/> <!-- em, strong, dfn, code, samp, kbd, span, abbr, math, a, img, br, q, sub, sup, bdo -->
+                <zeroOrMore>
+                    <choice>
+                        <ref name="inline"/> <!-- em, strong, dfn, code, samp, kbd, span, abbr, math, a, img, br, q, sub, sup, bdo -->
+                        <ref name="hr"/>
```

Comment:
#521 is about hr elements between code blocks. This would allow hr elements in an inline context.

Today it seems like hr elements are only allowed adjacent to a p element. See `<define name="p">`. I think this is because of how hr was marked up in DTBook, where we did it using a class on the p element, since there was no dedicated element for it.

Here in EPUB we could simply remove it from `<define name"p">` and add it to the list in `<define name="block">`. Then it would be allowed between codeblocks and verses.